main.py

```diff
@@ -198,10 +140,67 @@ async def on_member_join(self, member: discord.Member):
         await member.create_dm()
         await member.dm_channel.send(embed=embed)
 
-    async def on_member_update(self, before, after):
-        # Debug before
-        logger.debug(f"before | {before.roles}")
+    async def on_voice_state_update(self, member, before, after):
+        # Debug member.id
+        if debugMode: logger.debug(f"member.id | {member.id} {type(member.id)}")
```

Comment:
you can set up logger mode depending on your `DEBUG` const value
```suggestion
        # somewhere on the top of your code
        logging.getLogger().setLevel(
            logging.DEBUG if DEBUG_MODE else logging.INFO
        ) 
        # then you don't need to write ifs every time you want to debug something
        logger.debug(f"member.id | {member.id} {type(member.id)}")
```